### DISCUSSION TITLE
Suggestion: Game Mode Switching Menu

### DISCUSSION THREAD

**Discussion by Asveste** [Roadhog360/Et-Futurum-Requiem#140] [Ideas]
In 1.16 Mojang added a menu that lets you quickly change between all the game modes.
You could easily access it by pressing F3 + F4 and choose between every gamemode by pressing F4 again until you have the desired gamemode selected. Releasing the buttons now puts the player in their selected gamemode. If you open the menu again, the previously selected gamemode is automatically selected allowing you to quickly change between two gamemodes like survival and creative.
I attached an example image below.

![2022-07-05_04 28 55](https://user-images.githubusercontent.com/23436647/177237772-d960fe2d-6a67-409b-9290-71a04a09be27.png)

**Comment by Roadhog360** [comment]
This has been in several releases at this point thanks to @embeddedt
